cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/docker/docker-mc-common/mc-prerequisite-step/McPrerequisiteStep.tsx

```diff
@@ -17,14 +17,14 @@ function McPrerequisiteStep(props: Partial<StepProps>) {
     const [message, setMessage] = useState('');
     const connect = useCallback(async () => {
         setConnectionStatus(CONNECTION_STATUS.CONNECTING);
-        setMessage('Connecting to docker deamon');
+        setMessage('Verifying Docker deamon and resource allocations');
```

Comment:
Since you're modifying this anyway... isn't it `daemon`?

Reply:
Yes good catch, thank you